repos/system_upgrade/common/actors/scangrubdevice/actor.py

```diff
@@ -19,8 +19,5 @@ def process(self):
         if architecture.matches_architecture(architecture.ARCH_S390X):
             return
 
-        device_name = grub.get_grub_device()
-        if device_name:
-            self.produce(GrubInfo(orig_device_name=device_name))
-        else:
-            self.produce(GrubInfo())
+        devices = grub.get_grub_devices()
```

Comment:
@matejmatuska actually, as the `orig_device_name` is deprecated now, you should fill it also when only one grub device is set, otherwise it's not deprecation but removal with keeping dead fields present

Reply:
That raises a question, what should I put into orig_device_name if there is more than one device?

Reply:
Should I preserve the original behavior in such case? @pirat89

Reply:
In such a case i would keep it None as it was already incorrect. But in case of the one device, I would fill it.